src/app/features/schedule/schedule/schedule.component.ts

```diff
@@ -553,4 +638,68 @@ export class ScheduleComponent implements AfterViewInit, OnDestroy {
     parts[0] = `translate3d(${newNumber}`;
     return parts.join(',');
   }
+
+  getEventDayStr(ev: ScheduleEvent): string | null {
+    // Calendar events
+    if (ev.type === SVEType.CalendarEvent && ev.data && (ev.data as any).start) {
```

Comment:
Casting as `as any` is not good, since it leads to potential bugs and makes refactoring a lot harder.